### DISCUSSION TITLE
How to correctly delete `docs` from foam-template?

### DISCUSSION THREAD

**Discussion by theo-brown** [foambubble/foam#1288] [Q&A]
Hi,
What's the correct way of getting rid of the docs directory after cloning foam-template?
I deleted it in VSCode and now the graph is broken; it doesn't load, and raises the following error:
```
Error loading webview: Error: Could not register service worker: InvalidStateError: Failed to register a ServiceWorker: The document is in an invalid state..
```

**Comment by riccardoferretti** [comment]
There should be nothing in the `docs` directory that would impact the graph, I am not sure what to suggest except:
1. check that before deleting the docs directory everything works
2. check if deleting the docs folder recreates the issue
3. if it does, check if deleting the docs folder after having added a couple of new notes (outside of the dir of course, so that the repo is not empty) recreates the issue

I am just shooting in the dark here, I don't see a connection between your actions and the error you are getting
